docs/user-guide/deep_links.md

```diff
@@ -0,0 +1,57 @@
+# Deep Links
```

Comment:
Only administrators/operations will be able to configure deep links. So I suggest moving it to `operator-manual`.

Reply:
Hmm I saw configmanagement plugin doc in user guide so that that's where it will go, should I move that too?

Reply:
Thanks for noticing it! I think configmanagement should be moved as well.

Reply:
Moved both

Reply:
Heads up that this will probably break people's bookmarks